api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -121,6 +122,8 @@ message Transformation {
     // 'headers', containing the headers, and 'body', containing the original
     // body.
     HeaderBodyTransform header_body_transform = 2;
+    // Configuration for an externally implemented transformer.
```

Comment:
Any reason not to just have this API be in OSS and mark it as enterprise only. Would avoid the indirection

Reply:
this is api is not exposed to the user, we can do the OSS approach api in gloo to make it easy on the user.

I don't see an easy way to use this approach here - To do that we will essentially need to override the  transformation config factory from closed source, which i don't think is easy to do with envoy...